openqa-qam-sle

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I recommend

```suggestion
set -euo pipefail
```

to prevent bad surprises

Reply:
In my case this setting `set -o pipefail` is not appropriate because the pipe is expected to finish with a non-zero exit code, when it finds no match. In fact in most cases the return code of the pipe will not be zero.